internal/publish/config.go

```diff
@@ -98,6 +98,9 @@ type Config struct {
 	// Normally "still valid" keys can be accepted, but are embargoed.
 	ReleaseSameDayKeys      bool `env:"DEBUG_RELEASE_SAME_DAY_KEYS"`
 	DebugLogBadCertificates bool `env:"DEBUG_LOG_BAD_CERTIFICATES"`
+
+	// Publish stats API config
+	StatsUploadMinimum int64 `env:"STATS_UPLOAD_MINIMUM, default=10"`
```

Comment:
StatusUploadMinimum is the minimum number of statistics that must be present to return a response? It'd be good to document the behavior of what happens if the number is less than this.